engine-tests/src/tests/pausable_precompiles.rs

```diff
@@ -0,0 +1,121 @@
+use crate::prelude::U256;
+use crate::test_utils::standard_precompiles::{PrecompilesConstructor, PrecompilesContract};
+use crate::test_utils::{self, AuroraRunner, Signer, PAUSE_PRECOMPILES, RESUME_PRECOMPILES};
+use aurora_engine::parameters::{PausePrecompilesCallArgs, TransactionStatus};
+use aurora_engine_types::types::Wei;
+use borsh::BorshSerialize;
+
+#[test]
+fn test_executing_paused_precompile_throws_error() {
+    const MODEXP_FLAG: u32 = 0b000000000010000;
+    const TEST_MODEXP: &str = "test_modexp";
+    const CALLED_ACCOUNT_ID: &str = "aurora";
+
+    let (mut runner, mut signer, contract) = initialize_precompiles_contract();
+
+    let call_args = PausePrecompilesCallArgs {
+        paused_mask: MODEXP_FLAG,
+    };
+
+    let mut input: Vec<u8> = Vec::new();
+    call_args.serialize(&mut input).unwrap();
+
+    let _ = runner.call(PAUSE_PRECOMPILES, CALLED_ACCOUNT_ID, input.clone());
+    let (result, _) = runner
+        .submit_with_signer_profiled(&mut signer, |nonce| {
+            contract.call_method(TEST_MODEXP, nonce)
+        })
+        .unwrap();
+
+    let number = match result.status {
+        TransactionStatus::Succeed(number) => U256::from(number.as_slice()),
```

Comment:
I am confused about the outcome of this test. I thought after pausing trying to use the precompile should throw an error.